bom/pom.xml

```diff
@@ -226,6 +226,21 @@
         <artifactId>vertx-jca-adapter</artifactId>
         <version>${version.vertx}</version>
       </dependency>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-core</artifactId>
+        <version>${version.vertx}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-web</artifactId>
+        <version>${version.vertx}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-hazelcast</artifactId>
```

Comment:
Not trying to push anything, but I'd expect we'd utilize the Infinispan-based cluster manager :-)

Reply:
This one is only used for tests...